firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -142,6 +148,20 @@ public FirebaseOptions getOptions() {
     return options;
   }
 
+  /**
+   * Returns the specified {@link EmulatorSettings}.
+   *
+   * <p>TODO(samstern): Un-hide this once Firestore, Database, and Functions are implemented
+   *
+   * @hide
+   */
+  @KeepForSdk
+  @Nullable
```

Comment:
Would it make sense to make it `@NotNull` and return a default "empty" settings object. It should help with the following:
* Make it more null-safe and simplify integrating SDKs as they won't have to deal with `null` twice as I can see in the implementation below.
*iiuc it will help make the "set-after-use" logic below more safe, i.e. with the null case it seems possible to first initialize firestore and then call `enableEmulators()`

wdyt?